services/order.py

```diff
@@ -9,19 +11,17 @@
 
 
 def create_order(
-        tickets: list[dict],
-        username: str,
-        date: str = None
+    tickets: list[dict],
+    username: str,
+    date: Optional[str] = None
 ) -> Order:
     user = User.objects.get(username=username)
     new_data = {}
     if date:
         new_data["created_at"] = date

```

Comment:
isn't it better to create order here and if date -> set date? instead of creating this dict